rampwf/kits/base.py

```diff
@@ -0,0 +1,27 @@
+from os import environ, makedirs
+from os.path import expanduser, join, exists
+
+
+def get_data_home(ramp_kits_home=None):
```

Comment:
I would find it more logical to put this in utils as well, but not that important